tests/unit/package_managers/oci/test_oci_client.py

```
@@ -0,0 +1,154 @@
+            'Bearer realm="https://auth.docker.io/token",'
+            'service="registry.docker.io",'
+            'scope="repository:library/alpine:pull"'
+        )
+        realm, service, scope = parse_www_authenticate(header, "library/alpine")
+        assert realm == "https://auth.docker.io/token"
+        assert service == "registry.docker.io"
+        assert scope == "repository:library/alpine:pull"
+
+    def test_missing_scope_uses_default(self) -> None:
+        header = 'Bearer realm="https://auth.example.com/token",service="example.com"'
+        realm, service, scope = parse_www_authenticate(header, "org/repo")
```

Ruff (0.15.21)

[warning] Unpacked variable `service` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)